hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/interfaces/ChunkManager.java

```diff
@@ -106,6 +106,11 @@ default void finishWriteChunks(KeyValueContainer kvContainer,
     // no-op
   }
 
+  default void finalizeWriteChunk(KeyValueContainer container,
+      BlockData blockData) throws IOException {
+    // no-op
```

Comment:
there's no corresponding implementation within FilePerChunkStrategy, which is fine. I think that is a legacy code and doesn't make much sense for users to enable that configuration. But it would be easier to troubleshoot if it throws an exception instead of an no-op.